R/data_mile2.R

```diff
@@ -0,0 +1,38 @@
+#' FIMS input data frame for milestone 2
+#'
+#' A dataset containing information necessary to run an age-structured stock
+#' assessment model with length composition data in FIMS for milestone 2 . 
+#' This data was generated using
+#' the `ASSAMC` package written for the [model comparison project](
+#' www.github.com/Bai-Li-NOAA/Age_Structured_Stock_Assessment_Model_Comparison).
+#'
+#' @format A data frame with `r NROW(data_mile2)` observations of
+#' `r NCOL(data_mile2)` variables:
+#' \describe{
+#'   \item{type}{The type of data the row contains. Allowed types include
+#'     `age`, `length-comp`, `length-at-age`, `index`, `landings`, and `weight-at-age` data.}
+#'   \item{name}{A character string providing the name of the information source
+#'     that the data was collected from, e.g., `"Trawl fishery"`}
+#'   \item{age}{An integer age. Entry can be `NA` if information pertains to
+#'     multiple ages, e.g., total catch rather than catch of age-4 fish.}
+#'   \item{len}{An integer length. Entry can be `NA` if information pertains to
+#'     multiple lengths, e.g., total catch rather than catch of length-4 fish.
+#'     column is not required if no length data in the model.}
```

Comment:
"column ..." should be "This column is not required if the model is not fit to length data."

Reply:
I agree